### ISSUE TITLE
Stuck at Retrieving Stat information

### ISSUE THREAD

**Issue by JHH619** [gibbed/SteamAchievementManager#257]
So I launch the SAM and go to the game I want as usual, but where it usually loads the achievements, it just shows at the bottom "Retrieving Stat Information".

**Comment by rgrekoff** [comment]
Yes, I've got the same problem. This happens on a newest ver of steam client, build of Jan 16. SAM works just fine with a previous build of Dec 16. Something has obviously changed.

**Comment by fengmian309** [comment]
Have you solved this problem yet? I'm having the same problem.

**Comment by rgrekoff** [comment]
That's strange, I just turned on beta-ver participation in the steam settings and it solved the issue.

**Comment by ImpulsePOP** [comment]
> That's strange, I just turned on beta-ver participation in the steam settings and it solved the issue.

it didn't work for me(
